src/eth/mining.json

```
@@ -50,32 +50,32 @@
 	},
 	{
 		"name": "eth_submitWork",
-		"summary": "Used for submitting a proof-of-work solution.",
+		"summary": "Can be used by an external miner to submit their proof-of-work solution.",
 		"params": [
 			{
-				"name": "Proof-of-work hash",
+				"name": "nonce",
 				"required": true,
 				"schema": {
-					"$ref": "#/components/schemas/bytes32"
+					"$ref": "#/components/schemas/bytes"
 				}
 			},
 			{
-				"name": "seed hash",
+				"name": "hash",
 				"required": true,
 				"schema": {
 					"$ref": "#/components/schemas/bytes32"
 				}
 			},
 			{
-				"name": "difficulty",
+				"name": "digest",
 				"required": true,
 				"schema": {
 					"$ref": "#/components/schemas/bytes32"
 				}
 			}
 		],
 		"result": {
-			"name": "Success",
+			"name": "workWasAccepted",
```

```suggestion
			"name": "success",
```
I think success is appropriately clear.

---

reverted